### PR TITLE
feat: make navbar sticky

### PR DESCRIPTION
**Pull request by Tulun**
Created useScrollPosition hook, when SP > 0 add a border to the navbar
Screen Shot 2022-07-27 at 2 36 10 PM

Needs QA
Overview

Acceptance Criteria

How To Report QA
Testers, use the following templates to pass the pull request or to report issues that require a resolution. Following these guidelines will help the developer respond more quickly to issues, but you are free to comment in any way as needed to help resolve issues.

Reporting QA Issues

Testers should use Github comments to report issues with the PR. Use screenshots if relevant, you can also check the [developer console](https://support.airtable.com/hc/en-us/articles/232313848-How-to-open-the-developer-console) to see if there are any logs which might be related. Use the following template when reporting a problem or issue with the PR. You submit multiple comments using this template if there are multiple issues.

**QA Issue**
1. Navbar stays stick when you can scroll down.
2. Border bottom is added.
3. Responsive.

**Repro Steps**
Be able to scroll viewport.

### CODE REVIEW

File: src/components/Header/Header.styles.tsx [hunk 1]
```diff
@@ -2,13 +2,23 @@ import styled from "@emotion/styled";
...
   background-color: #2d2e33;
   height: 72px;
   padding: 0 24px;
   display: flex;
   align-items: center;
   color: #c5d5e0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
```
Comment (amateima) [inline src/components/Header/Header.styles.tsx:18]: Let's put in the README file the z-index values we use for various components: headers, tooltips, pup-ups, toasts, etc (I think it's only the header for now). We'll add pop-ups in the near future and we'll have to guess the z-index values to make them positioned on top of everything

**Review by amateima — CHANGES_REQUESTED** [review]
All the content across the web app is being cut now due to the new `fixed` positioning

Screenshot 2022-07-28 at 11 50 37
Screenshot 2022-07-28 at 11 51 03

For example, the top padding of this wrapper should include the header height too.

Screenshot 2022-07-28 at 11 52 34

File: src/components/Header/Header.styles.tsx [hunk 2]
```diff
@@ -2,13 +2,23 @@ import styled from "@emotion/styled";
...
+  border-bottom: ${({ scrollPosition }) => {
+    return scrollPosition > 0 ? "1px solid #4d4f56" : "none";
```
Comment (amateima) [inline src/components/Header/Header.styles.tsx:20]: I think this should be `scrollPosition > 0 ? "1px solid #4d4f56" : "1px solid transparent"`. Making it `none` decreases the height of the header content on scroll which produces an annoying effect

**Review by amateima — CHANGES_REQUESTED** [review]

**Comment by daywiss** [comment]
running into some of the same issues as alex, will re-test once those are addressed

File: src/components/Header/Header.styles.tsx (hunk 2 above)
Comment (Tulun) [reply src/components/Header/Header.styles.tsx:20]: Fair, changed.

File: src/components/Header/Header.styles.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/Header/Header.styles.tsx:18]: Added.

**Comment by Tulun** [comment]
> running into some of the same issues as alex, will re-test once those are addressed

@daywiss feel free to review, fixed the issues.

**Comment by Tulun** [comment]
@daywiss @amateima also changed the display to ref and copy to ref as that was requested and it's a very minor change.

**Comment by daywiss** [comment]
im seeing a gap in the footer when the browser is at certain heights 
image

**Comment by Tulun** [comment]
> im seeing a gap in the footer when the browser is at certain heights image
> 
> this is at roughly 1000px height

haha just fixed this.

**Review by daywiss — APPROVED** [review]
lgtm

**Comment by amateima** [comment]
> @daywiss @amateima also changed the display to ref and copy to ref as that was requested and it's a very minor change.

you can make it as part of a separate PR, but the referral link box needs some minor changes. [See here](https://linear.app/uma/issue/ACX-90/add-ref-as-supported-query-parameter-for-the-referrer-address) (4 address characters, the container box width, the URL button left/right padding)

**Review by amateima — APPROVED** [review]

**Pull request merged**